code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -442,10 +443,10 @@
 	antag_flag = ROLE_BLOB
 	required_enemies = list(2,2,1,1,1,1,1,0,0,0)
 	required_candidates = 1
-	minimum_round_time = 35 MINUTES
-	weight = 3
-	cost = 8
-	minimum_players = 25
+	minimum_round_time = 45 MINUTES
+	weight = 4
+	cost = 10
+	minimum_players = 40
```

Comment:
This makes blob unlikey to _ever_ spawn.

Reply:
minimum_players counts DS2 and interlink players, which isn't good while people are not wanting to interact with the blob at all, which resorts to either having a complicated system of keeping track of players who WANT to fight the blob, and players who don't want any of it.
Blobs don't have any challenge at all on this server, and the only way for the crew to beat it with such a low fighting population is rough, which isn't fun for either side.
The server at this current time of posting as 52 players, which is enough for a blob to spawn.
These are changes in preparation for 1.0, which means we will go on the HUB, the blob will spawn, then.

Lastly, blob spawning in the first place was for testing purposes, We have had enough blob rounds to test the crew, let's prep for 1.0

Reply:
> DS2 and interlink players

DS-2 is removed.

> The server at this current time of posting as 52 players

This implies all 52 players play, and don't observe.

Reply:
Blob requires a decently high playercount so this is fine honestly.

Reply:
> This makes blob unlikey to _ever_ spawn.

Yes, and that's a good thing in my opinion, the blob tends to steamroll things with our pop.